changelog/duplicate-implementations-deprecation.dd

```diff
@@ -0,0 +1,26 @@
+Usage of identical functions in a single module
```

Comment:
```suggestion
A deprecation is now issued on multiple definitions of identical functions within the same module
```

Rationale: Usually we say "<Feature> has be deprecated" or "Deprecated the ability to <feature>".
However, the previous code was buggy, hence the formulation "A deprecation is now issued"